ReadME.md

```diff
@@ -79,4 +79,4 @@ serve -s build -l 4000
 
 Now you should be able to access the Netflix clone application by opening your web browser and navigating to `http://localhost:4000`.
 
-Enjoy exploring the Netflix clone!!
```

Comment:
missed dots

Reply:
thanks for identifying. Will merge the PR